main.tex

```diff
@@ -247,7 +247,7 @@ \subsection*{\textit{Vgsc} non-synonymous nucleotide variation}
 %
 The \texttt{N1570Y} variant was present in Guinea, Burkina Faso (both species) and Cameroon.
 %
-This variant has only ever been found in \textit{An. gambiae} in association with \texttt{L995F} \cite{Jones2012}, and has been shown experimentally to substantially increase pyrethroid resistance when it occurs in combination with \texttt{L995F} \cite{Wang2015}.
+This variant has only ever been found in \textit{An. gambiae}/\textit{An. coluzzii} in association with \texttt{L995F} \cite{Jones2012}, and has been shown experimentally to substantially increase pyrethroid resistance when it occurs in combination with \texttt{L995F} \cite{Wang2015}.
```

Comment:
I think we could just delete "An. gambiae/An. coluzzii" altogether, so it becomes:

"...only ever been found in association with L995F..."

Reply:
yep